robots/image.js

```diff
@@ -0,0 +1,40 @@
+const state = require('./state.js')
+const google = require('googleapis').google
+const customSearch = google.customsearch('v1')
+const googleSearchCredentials = require('../credentials/google-search.json')
+
+
+async function robot() {
+    const content = state.load()
+    await fetchImagesOfAllSentences(content)
+    state.save(content)
+
+    async function fetchImagesOfAllSentences(content) {
+        for (const sentence of content.sentences) {
+            const query = `${content.searchTerm} ${sentence.keywords[0]}`
+            sentence.images = await fetchGoogleAndReturnImagesLinks(query)
+
+            sentence.googleSearchQuery = query
+        }
+    }
+
+
+    async function fetchGoogleAndReturnImagesLinks(query) {
+        const response = await customSearch.cse.list({
+            auth: googleSearchCredentials.apiKey,
+            cx: googleSearchCredentials.searchEngineId,
+            q: query,
+            searchType: 'image',
+            num: 2
```

Comment:
You could add the parameter `imgSize: xlarge` to assure you'll get images with good enough quality, but not too big

https://developers.google.com/custom-search/v1/cse/list